Loop/Managers/WatchDataManager.swift

```diff
@@ -40,6 +43,52 @@ final class WatchDataManager: NSObject {
     }()
 
     private var lastSentSettings: LoopSettings?
+    
+    let healthStore: HKHealthStore
```

Comment:
I believe we no longer need a HKHealthStore instance directly inside WatchDataManager, since the access is managed through SleepStore; is this property used anywhere?

Reply:
Nope, removed